src/Test.UnitTests.Sarif.Multitool.Library/RewriteCommandTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class RewriteCommandTests
+    {
+        private static readonly ResourceExtractor Extractor = new ResourceExtractor(typeof(RewriteCommandTests));
+
+        [Fact]
+        public void RewriteCommand_WhenOutputFormatOptionsAreInconsistent_Fails()
+        {
+            const string SampleFilePath = "minimal.sarif";
+            File.WriteAllText(SampleFilePath, Extractor.GetResourceText($"RewriteCommand.{SampleFilePath}"));
```

Comment:
>WriteAllText [](start = 17, length = 12)

Are we writing a file to disk simply to validate that a command-line is invalid? I suggest we not do this. i.e., refine this test so that it doesn't have any IO (which it doesn't require in order to provide its test value).